.rubocop.yml

```diff
@@ -12,6 +12,11 @@ inherit_from: .rubocop_todo.yml
 Layout/IndentHeredoc:
   Enabled: false
 
+Metrics/ClassLength:
```

Comment:
I think we should disable this one altogether or raise the max for everything.

Reply:
Yeah I didn't want to be overzealous here 😆

Reply:
You should make this change. This excludes files from ClassLength specifically and unnecessarily.

Reply:
This is resolved in https://github.com/intridea/hashie/pull/479/commits/9f4b47a259b69aeb925ba08c63ded1c74ad4c127